guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,70 @@
+[id="migrating_pulp_content"]
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content and the number of content views.
+For large systems, this can mean several days of downtime.
+To prevent this, {Team} recommends that you pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
+This reduces the overall upgrade downtime.
+
+Use this procedure to begin migrating content from Pulp 2 to Pulp 3.
+
+.Procedure
+. View details of the content you are migrating using the following command:
```

Comment:
Is this procedure a pre-migration or a migration? I am confused.

Reply:
~migrating~ --> pre-migrating